# Special course: Isolation, sequencing and assembly of _streptomyces_ from soil
The aim of the project is to isolate, sequence and assemble genomes of actinomycete strains derived from soil. As such, I expect we will perform the following lab-work:
- Isolation of actinomycetes from soil (should be covered by general microbiological background)
- Extraction of HMW DNA
- QC of DNA quality
- Oxford Nanopore Sequencing

In addition to the lab work, the course will involve the following analysis work:
- Genome assembly
	- Using Flye (and perhaps Trycycler) to generate genome assemblies
	- Annotating assemblies using Bakta
	- Evaluating the quality of assemblies using BANDAGE and BUSCO
- Structuring analysis work using scripts
	- Navigating a terminal and automating small tasks using BASH (BASHCRAWL could be a fun intro)
	- Plotting using R/Python

# Relevant Literature
- Wick, Ryan R., et al. “Assembling the Perfect Bacterial Genome Using Oxford Nanopore and Illumina Sequencing.” Plos Computational Biology, edited by Francis Ouellette, vol. 19, no. 3, 2023, p. e1010905, https://doi.org/10.1371/journal.pcbi.1010905. 
- Schwengers, Oliver, et al. “Bakta: Rapid and Standardized Annotation of Bacterial Genomes via Alignment-Free Sequence Identification.” Microbial Genomics, vol. 7, no. 11, 2021, p. 000685, https://doi.org/10.1099/MGEN.0.000685.
- Simão, Felipe A., et al. “BUSCO: Assessing Genome Assembly and Annotation Completeness with Single-Copy Orthologs.” Bioinformatics, vol. 31, no. 19, 2015, pp. 3210–12, https://doi.org/10.1093/bioinformatics/btv351.
- Alvarez-Arevalo, Maria, et al. “Extraction and Oxford Nanopore Sequencing of Genomic DNA from Filamentous Actinobacteria.” Star Protocols, vol. 4, no. 1, 2023, p. 101955, https://doi.org/10.1016/j.xpro.2022.101955. 
